Cartfile.resolved

```diff
@@ -1,8 +1,8 @@
-github "amplitude/Amplitude-iOS" "a3d6720626e72a678489bce83ed9dacadb505dd7"
+github "amplitude/Amplitude-iOS" "32d36977147d9f9a6330002afdc85fe31cfbcfaa"
```

Comment:
Reverted.  It's failing travis because the binaries aren't checked in.  Would you like me to commit those?  I'm not sure how much this kind of activity grows the repo.